doc/howto/autocompletion.md

```diff
@@ -32,6 +32,12 @@ conda create --name jupysqlenv python=3.10 jupyterlab --yes --channel conda-forg
 conda activate jupysqlenv
 ```
 
+This feature requires Node, if you don't have it, you can install it with conda:
+
+```sh
+conda install --channel conda-forge nodejs --yes
+```
```

Comment:
I believe node is only required for building the extension, right? It isn't needed for an end-user. If so, this should be part of the autocompletion package developer documentation

Reply:
yeah you're right. Removed it from here.
It's mentioned in the [developer doc](https://github.com/ploomber/jupysql-autocompletion/blob/main/README.md)